source/linux/Foundational_Components/Kernel/Kernel_Drivers/Camera/CSI2RX.rst

```diff
@@ -710,6 +710,29 @@ Enabling camera sensors
     For more details on building or applying overlays permanently, refer to the
     :ref:`How to enable DT overlays in linux <howto_dt_overlays>` guide.
 
+
+    CSI2RX testing details
```

Comment:
I would prefer to headline it as Supported CSI2x sensors instead of testing details.